CONTRIBUTORS.rst

```diff
@@ -2,17 +2,20 @@
     place names and roles of the people who contribute to this package
     in this file, one to a line, like so:
 
-- Timo Stollenwerk, Original Author
```

Comment:
@arsenico13 we usually just append names at the end as a best practice in the Plone community. I appreciate that you want to sort the authors by name but this never really worked. Most people add their names at the end, so I think we should keep things simple and just always append the names at the end...

Reply:
Sorry, didn't know that!
Restored the old order and added missing contributors!